isotptun.c

```diff
@@ -154,9 +177,9 @@ int main(int argc, char **argv)
 			else if (elements == 2)
 				opts.flags |= (CAN_ISOTP_EXTEND_ADDR | CAN_ISOTP_RX_EXT_ADDR);
 			else {
-				printf("incorrect extended addr values '%s'.\n", optarg);
+				fprintf(stderr, "incorrect extended addr values '%s'.\n", optarg);
 				print_usage(basename(argv[0]));
-				exit(0);
+				exit(EXIT_FAILURE);
 			}
```

Comment:
The change on error -> `exit(EXIT_FAILURE)` and the non functional change `exit(0)` to `exit(EXIT_SUCCESS)` can go into a single patch.